.github/workflows/pr-build-smoke.yml

```
@@ -0,0 +1,136 @@
+  build_bunkbot:
+    name: Build + Smoke (bunkbot)
+    needs: changes
+    if: needs.changes.outputs.bunkbot == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build image
+        run: |
+          docker build -f apps/bunkbot/Dockerfile -t bunkbot:pr-${{ github.event.number }} .
+      - name: Smoke start container
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2034:warning:6:1: i appears unused. Verify use (or export if used externally)

(shellcheck)